test/expect/TestONNX.test_basic.expect

```diff
@@ -35,7 +35,43 @@ graph {
     }
   }
   name: "torch-jit-export"
-  input: "1"
-  input: "2"
-  output: "7"
+  input {
+    name: "1"
+    type {
+      tensor_type {
+        elem_type: FLOAT
+        shape {
+          dim {
+            dim_value: 1
```

Comment:
Heh, just looking at this proto, it's so disappointing that we had to support symbolic dimensions :o)